sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -260,6 +260,13 @@ object RapidsConf {
     .checkValue(v => v >= 0 && v <= 1, "The fraction value must be in [0, 1].")
     .createWithDefault(0.9)
 
+  val RMM_ALLOC_MAX_FRACTION = conf("spark.rapids.memory.gpu.maxAllocFraction")
+      .doc("The fraction of total GPU memory that limits the maximum size of the RMM pool. " +
+          "If this is set to 0 then the pool size will not be artificially limited.")
+      .doubleConf
+      .checkValue(v => v >= 0 && v <= 1, "The fraction value must be in [0, 1].")
+      .createWithDefault(0)
```

Comment:
`createWithDefault(1)` shouldn't that naturally describe the "do not limit" case? Also if I set `RMM_ALLOC_FRACTION` to 0, it seems `RMM_ALLOC_MAX_FRACTION` should be 0 also?